Project.toml

```diff
@@ -0,0 +1,8 @@
+name = "LearnBase"
+uuid = "7f8f8fb0-2700-5f03-b4bd-41f8cfc144b6"
+repo = "https://github.com/JuliaML/LearnBase.jl.git"
+version = "0.2.0"
```

Comment:
I'm not sure about this ... maybe remove the version line so that it doesn't conflict with new tags?

Reply:
I thiught Pkg3 uses this line instead of tags to figure out dependency versions, doesn't it? 

In a couple of my 0.7-ready packages I just set the same number in attobot, but maybe there's a smarter way.